docs/docs/concepts/memory.md

```diff
@@ -160,9 +160,77 @@ config = {"configurable": {"thread_id": "1"}}
 graph.get_state(config)
 ```
 
-Persistence is critical sustaining a long-running chat sessions. For example, a chat between a user and an AI assistant may have interruptions. Persistence ensures that a user can continue that particular chat session at any later point in time. However, what happens if a user initiates a new chat session with an assistant? This spawns a new thread, and the information from the previous session (thread) is not retained. This motivates the need for a memory service that can maintain data across chat sessions (threads). 
+Persistence is critical sustaining a long-running chat sessions. For example, a chat between a user and an AI assistant may have interruptions. Persistence ensures that a user can continue that particular chat session at any later point in time. However, what happens if a user initiates a new chat session with an assistant? This spawns a new thread, and the information from the previous session (thread) is not retained. This motivates the need for a memory that can maintain data across chat sessions (threads). 
 
-## Meta-prompting
+For this, we can use LangGraph's `Store` interface to save and retrieve information across threads. Shared information can be namespaced by, for example, `user_id` to retain user-specific information across threads. Let's show how to use the `Store` interface to save and retrieve information.
+
+```python
+from langgraph.store.memory import InMemoryStore
+in_memory_store = InMemoryStore()
+
+# Namespace for memories
+user_id = "1"
+namespace_for_memory = (user_id, "memories")
+
+# Save memories 
+memory_id = str(uuid.uuid4())
```

Comment:
If there's a single sentence answer  to the question why is the key not a tuple? Why has the key been broken into a namespace (tuple) and a string.

Reply:
Kinda like files and folders, but there's not a super strong distinction in some cases